docker_sandbox/API/payload/script.sh

```diff
@@ -38,21 +38,21 @@ addtionalArg=$4
 #
 ########################################################################
 
-exec  1> $"/usercode/logfile.txt"
-exec  2> $"/usercode/errors"
+exec  1> $"logfile.txt"
+exec  2> $"errors"
 #3>&1 4>&2 >
 
 START=$(date +%s.%2N)
 #Branch 1
-if [ "$output" = "" ]; then
-    $compiler /usercode/$file -< $"/usercode/inputFile" #| tee /usercode/output.txt
+if [[ "$output" = "" ]]; then
+    ${compiler} ${file} -< $"inputFile" #| tee /usercode/output.txt
 #Branch 2
 else
 	#In case of compile errors, redirect them to a file
-        $compiler /usercode/$file $addtionalArg #&> /usercode/errors.txt
+        ${compiler} ${file} ${addtionalArg} #&> /usercode/errors.txt
```

Comment:
remove addtionalArg
decide on an argument order that makes since - additional args are part of the compilation line itself, output and input should come after

Reply:
changed the order of the args - now additionalArgs will be the 3rd argument while the output file to be executed is the 4th and last arg